arcade/context.py

```diff
@@ -311,10 +311,7 @@ def reset(self) -> None:
         self.bind_window_block()
         # self.active_program = None
         self.viewport = 0, 0, self.window.width, self.window.height
-        self.view_matrix = Mat4()
-        self.projection_matrix = Mat4.orthogonal_projection(
-            0, self.window.width, 0, self.window.height, -100, 100
-        )
+        self._default_camera.use()
         self.enable_only(self.BLEND)
         self.blend_func = self.BLEND_DEFAULT
         self.point_size = 1.0
```

Comment:
The reset method was more for unit tests. We can still probably trim it down. These lines should definitly not be called every frame but they should be called in conftest between tests except that blending needs to disable instead.

I also don't know if it's necessary to bind the windowblock every frame but that should be done in unit tests.